### ISSUE TITLE
I think this is wrong in "Explain how prototypal inheritance works"

### ISSUE THREAD

**Issue by chenjigeng** [yangshun/front-end-interview-handbook#208]
When a property is accessed on an object and if the property is not found on that object, the JavaScript engine should looks at the object's **\_\_proto\_\_**(different browser has their own implement) .Or, looks at the prototype of its constructor.

**Comment by sezgi** [comment]
Agreed. "All JavaScript objects have a `prototype` property" should say "All JavaScript objects have a `__proto__` property."

**Comment by funkybunky** [comment]
Actually, not *all* JS objects have a prototype. You can run the following:
```
let a = Object.create(null) // special case
console.log(a)
console.log(a.__proto__) // undefined
let b = {} // normal object
console.log(b)
console.log(b.__proto__)
```

**Comment by chenjigeng** [comment]
Yes.The first para of Object.create is proto. If you provide a null, it will cause this.

**Comment by yangshun** [comment]
Thanks for your suggestions, we've fixed it in d24b5d8
